app/templates/briefs/view_response_result.html

```diff
@@ -64,48 +64,51 @@
     <div class="dmspeak">
 
     {% if result_state == "submitted_ok" %}
-      <h2 class="summary-item-heading">What happens next</h2>
-
-      {% if 'essentialRequirementsMet' in brief_response %}
-        <h3>Shortlist</h3>
-        
-        <p>When the opportunity closes, the buyer will score your evidence. If you’re one of the top {{ brief.get('numberOfSuppliers') }} suppliers, you'll go through to the evaluation stage.</p>
-        <p>If you’re not successful, the buyer will give you feedback on the evidence you provided.</p>
-
-        <h3>Evaluation</h3>
-
-        <div class="explanation-list">
-          <p class="lead">At the evaluation stage, the buyer will ask you to provide:</p>
-          <ul class="list-bullet">
-            <li>
-            {% if brief.lotSlug == "digital-specialists" %}
-              evidence of the specialist’s skills and experience
-            {% else %}
-              evidence of your skills and experience
-            {% endif %}
-            </li>
-            {% if brief.lotSlug != "digital-specialists" %}
-            <li>your proposal</li>
-            {% endif %}
-          </ul>
-        </div>
-        <div class="explanation-list">
-          <p class="lead">The buyer will use the assessment methods listed in their requirements to evaluate your evidence. They’ll use:</p>
-          <ul class="list-bullet">
-            {% for eval_type in brief_summary.get_question('evaluationType').value %}
-              <li>{{ 'an' if eval_type == 'Interview' else 'a' }} {{ eval_type|lower }}</li>
-            {% endfor %}
-          </ul>
-        </div>
-        <p>
-          Your evidence must describe the skills and experience of the {{ 'person' if brief.lotSlug == "digital-specialists" else 'people' }} who’ll be working on the project.
-        </p>
-        <p>
-          The buyer will score all suppliers who reached the evaluation stage using the weightings they published with their requirements. They’ll provide feedback if you’re unsuccessful.
-        </p>
-      {% else %}
-        {% include 'briefs/_legacy_view_response_result_content.html' %}
-      {% endif %}
+      <div class="grid-row" data-analytics="trackPageView" data-url={{ "/opportunities/{}/responses/result/?result=success".format(brief.id) }}>
```

Comment:
If I recall, that weird flash from the view itself was a thing added by Tom specifically for triggering analytics events only in the right circumstances - I don't think it's used for displaying a message on the page at all (at least, it shouldn't be?).